packages/components/src/__future__/Tag/RemovableTag/subcomponents/RemoveButton.module.scss

```diff
@@ -0,0 +1,96 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "../../../../../styles/button-reset";
+
+.removeButton {
+  @include button-reset;
+
+  position: relative;
+  display: inline-flex;
+  border-radius: 50%;
+
+  &:focus {
+    outline: none;
+  }
+
+  &:focus-visible {
+    &::after {
+      $focus-ring-offset: -3px;
+
+      border-color: $color-blue-500;
+      content: "";
+      position: absolute;
+      background: transparent;
+      border-width: $border-focus-ring-border-width;
+      border-style: $border-focus-ring-border-style;
+      border-radius: 50%;
+      inset: $focus-ring-offset;
+    }
+  }
+
+  &::before {
+    position: absolute;
+    content: "";
+    inset: -5px;
+    border-radius: 50%;
+  }
+
+  .closeIcon {
+    fill: rgba($color-purple-800-rgb, 0.7);
+
+    &:hover,
+    &:focus-visible {
+      cursor: pointer;
+      fill: $color-purple-800;
+    }
+  }
+}
+
+.gray {
+  &:hover,
+  &:focus-visible {
+    background-color: rgba($color-gray-600-rgb, 0.2);
+  }
+}
```

Comment:
Optional: but I found this neat trick using locally scoped CSS variables that would cut down the repetition and slim down the CSS even more:
```
.removeButton {
  --hover-bg-color: rgba($color-gray-600-rgb, 0.2);
  ...
  &:hover,
  &:focus-visible {
    background-color: var(--hover-bg-color);
}

.blue {
  --hover-bg-color: rgba($color-blue-500-rgb, 0.3);
}
```

Which gets you down to a ~81ch line of CSS per colour
```
.RemoveButton-module_blue__tGSQ3{--hover-bg-color:rgba($color-blue-500-rgb,0.3)}
```

Reply:
Yeah cool that's pretty neat. We're getting rid of colors for now but this would be good to add in if we bring them back.